_drafts/SC2-5-3-label-content-name-mismatch.md

```diff
@@ -0,0 +1,103 @@
+---
+name: label and name from content mismatch
+description: |
+  Interactive elements labelled through their content must have their visible label as part of their [accessible name](#accessible-name)
+
+success_criterion:
+- 2.5.3 # Label in Name
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+- Bryn Anderson
+- Jey Nandakumar
+- Wilco Fiers
+---
+
+## Test procedure
+
+### Applicability
+
+This rule applies to any element that has:
+* a [widget role](https://www.w3.org/TR/wai-aria-1.1/#widget_roles) that [supports name from content](https://www.w3.org/TR/wai-aria-1.1/#namefromcontent), and 
+* [visible text content](#visible-text-content), and
```

Comment:
this definition will be tackled in - https://github.com/auto-wcag/auto-wcag/issues/143

Reply:
Interested to learn why this is different from "text that is [visible on the page][]"